.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+**Description**
+
+*Include a high-level description of the feature or error here including steps
+of how to recreate it if applicable. Include any benefits, challenges, or
```

Comment:
Consider replacing "including steps of how to ..." with "including steps for recreating it, if applicable." (Stylistic change.)